Create guest_net_timeout variable.

jcsmith opened the pull request:
When using this module to deploy within our infrastructure we
encountered time out errors while waiting on the deployment of the AVI
appliances.

Increasing the value of the wait_for_guest_net_timeout attribute of the
vsphere_virtual_machine.avi_controller resource from the default value
of 1 to 2 solved this problem for us.

This change introduces the guest_net_timeout variable to the module with
to allow this attribute to be configured on a case-by-case basis.  The
default value is still 1.

Commit:
Create guest_net_timeout variable.

When using this module to deploy within our infrastructure we
encountered time out errors while waiting on the deployment of the AVI
appliances.

Increasing the value of the wait_for_guest_net_timeout attribute of the
vsphere_virtual_machine.avi_controller resource from the default value
of 1 to 2 solved this problem for us.

This change introduces the guest_net_timeout variable to the module with
to allow this attribute to be configured on a case-by-case basis.  The
default value is still 1.

slarimore02 reviewed (changes requested):
Hey Joshua thank you for submitting a pull request for this issue. I think the way I'd like to resolve this is by removing the wait_for_guest_net_timeout = "1" line completely so the default is used. The default value is 5 minutes which should be fine for most environments

jcsmith commented:
That sounds reasonable to me as well. I'll make the changes in my fork and get those pushed later today.

Commit:
Revert "Create guest_net_timeout variable."

This reverts commit 2ac9fa925b73629eb914b4c00f793f292b10dbb9.

Commit:
Remove the wait_for_guest_net_timeout attribute.

After discussion it was decided the better way to handle this situation
was to remove the wait_for_guest_net_timeout attribute and instead rely
on the provider default of 5 minutes.
